corehq/apps/hqwebapp/session_details_endpoint/views.py

```diff
@@ -69,6 +72,14 @@ def post(self, request, *args, **kwargs):
             domains.update(EnterprisePermissions.get_domains(member_domain))
 
         enabled_toggles = toggles_enabled_for_user(user.username) | toggles_enabled_for_domain(domain)
+        enabled_previews = previews_enabled_for_domain(domain)
+        end_time = datetime.now()
+        metrics_histogram("commcare.session_details.processing_time",
+                      int((end_time - start_time).total_seconds() * 1000),
+                      bucket_tag='duration_bucket',
+                      buckets=(100, 250, 500, 1000, 2000, 3000, 4000),
+                      bucket_unit='ms',
+                      tags={'domain': domain})
```

Comment:
Would you mind throwing in `limit_domains` here?
https://github.com/dimagi/commcare-hq/blob/f51b264bfbe167b0440e67da464639bfa7e54131/corehq/util/metrics/__init__.py#L317-L321

Reply:
[fa1885c](https://github.com/dimagi/commcare-hq/pull/33955/commits/fa1885c1e5b3bd5ee052a27213b9e451700610a6)